### PR TITLE
feat!: allow processing to be disabled (resolves #174)

### LINKED ISSUE #174: Allow CSS, Sass, and/or JS processing to be disabled
## Is your feature request related to a problem?

CSS, Sass, and/or JS processing should be able to be easily disabled for projects which don't require them.

## Describe the solution you'd like

Options object should have appropriate keys for CSS, Sass, and JS which can be set to `false` to disable the functionality.

## Describe alternative solutions you've considered

Not applicable.

## Additional context or notes

Not applicable.

### PR DESCRIPTION
**Pull request by greatislander**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,27 +125,40 @@ module.exports = function (config) {
...
-    "basePath": "./src/assets/styles",
-    "minify": true,
-    "sourceMap": false,
-    "drafts": {
-        "nesting": true
+```js
+let options = {
+    /* Where should Eleventy look for CSS files to process? */
+    basePath: "./src/assets/styles",
+    /* Should CSS files be processed? */
```
Comment (jobara) [inline README.md:144]: Should these comments say "Sass" instead of "CSS"?

File: README.md [hunk 2]
```diff
@@ -99,27 +125,40 @@ module.exports = function (config) {
...
+    enabled: true,
```
Comment (jobara) [inline README.md:145]: From the index.js file it looks like this is disabled by default.

Comment (greatislander) [reply README.md:145]: Correct— it's not enabled by default.

Comment (jobara) [reply README.md:145]: It looks like this documents the default configuration for Sass compilation and that line 145 should be:

```suggestion
    enabled: false,
```

File: README.md [hunk 3]
```diff
@@ -68,28 +68,54 @@ module.exports = function (config) {
...
+module.exports = function (config) {
+-    config.addPlugin(fluidPlugin);
++    config.addPlugin(fluidPlugin, {
++        css: {
++              enabled: false,
++        }
++        sass: {
++              enabled: true,
++        }
++    });
```
Comment (jobara) [inline README.md:114]: Formatting errors in the options object.

```suggestion
+    config.addPlugin(fluidPlugin, {
+        css: {
+              enabled: false
+        },
+        sass: {
+              enabled: true
+        }
+    });
```

File: README.md [hunk 4]
```diff
@@ -99,27 +125,40 @@ module.exports = function (config) {
 -    config.addPlugin(fluidPlugin);
 +    config.addPlugin(fluidPlugin, {
 +        css: {
++              enabled: false,
++        },
++        sass: {
 +              browserslist: '> 1.5%',
++              enabled: true,
 +        }
 +    });
```
Comment (jobara) [inline README.md:134]: Linting options:

```suggestion
+    config.addPlugin(fluidPlugin, {
+        css: {
+              enabled: false
+        },
+        sass: {
+              browserslist: '> 1.5%',
+              enabled: true
+        }
+    });
```

**Review by jobara — APPROVED** [review]

**Pull request merged**